### ISSUE TITLE
[BUG] TokenStream.onComplete can not get useage

### ISSUE THREAD

**Issue by wzglgo** [langchain4j/langchain4j#1390]
- LangChain4j version:0.31.0
![image](https://github.com/langchain4j/langchain4j/assets/95947852/303aa364-79c0-4331-96f0-5a0923b45996)

model:azure chatgpt4o

**Comment by wzglgo** [comment]
```java
public interface ChatAssistant {
//    TokenStream streamingChat(@MemoryId String memoryId, @UserMessage String userMessage);
//    String chat(@MemoryId int memoryId, @UserMessage String userMessage);
    TokenStream streamingChat(String userMessage);
    String chat(String userMessage);
}

ChatAssistant assistant = AiServices.builder(ChatAssistant.class)
                .streamingChatLanguageModel(streamingChatModel)
                .chatMemory(chatMemory)
                .build();
        TokenStream tokenStream = assistant.streamingChat(content);
        tokenStream.onNext(ret -> {
            try {
                emitter.send(SseEmitter.event().data(ret));
            } catch (IOException e) {
                emitter.completeWithError(e);
                e.printStackTrace();
            }
        }).onComplete(ret -> {
            try {
                emitter.send(SseEmitter.event().data("[stop]"));
                emitter.complete();
                chatDetailLogService.afterCompleted(requestId, chatRoleEntity, LoginContextHolder.get().getAccountId() ,ret, content);
            } catch (IOException e) {
                emitter.completeWithError(e);
                e.printStackTrace();
            }
        }).onError(err -> {
            try {
                emitter.send(SseEmitter.event().data("[stop]"));
                emitter.completeWithError(err);
            } catch (IOException e) {
                emitter.completeWithError(e);
                e.printStackTrace();
            }
        }).start();`
```

**Comment by langchain4j** [comment]
Fixed by https://github.com/langchain4j/langchain4j/pull/1282
Will be released tomorrow in 0.32.0
